### DISCUSSION TITLE
otool_xtb script still needed for ORCA 5.0.3?

### DISCUSSION THREAD

**Discussion by SchroedingersChinchilla** [grimme-lab/xtb#708] [Q&A]
Hi all,

I downloaded the otool_xtb script https://xtb-docs.readthedocs.io/en/latest/setup.html#using-xtb-with-orca indicated here and moved it into my ORCA directory.  I ran a simple test job in ORCA and the output file indicated that it couldn't find otool_xtb, and that it would use xtb itself instead.  The run terminated normally, without throwing any errors, though the calculated energy is incorrect.  Is that expected (since the purpose of GFN-xTB is geometry, frequency, and non-covalent interactions)?  Is there something else that I need to be doing?

Thanks in advance!

**Comment by haneug** [comment]
The command in the docs simply creates a symlink (called otool_xtb) that points to your xtb binary. So using the xtb binary directly is not a problem. What do you mean by the calculated energy being incorrect? Are you looking at total energies or relative energies? Note that GFN also means that energies were not included in the parametrization and they can sometimes be off by a lot (for example if you calculate atomization energies). An explicit example would be helpful to understand your problem!

**Reply by SchroedingersChinchilla** [reply]
I see, thanks for the reply!  I ran a single-point energy calculation on C60 just to see if the program was working correctly, and I got a total energy of -128 Hartrees.  I ran the same molecule with r2SCAN-3c and got -2286 Hartrees.  If the energy values generated by xtb are just nonsense and not the aim of the program, that's understandable, but I just wanted to check if that's the case or not.

**Reply by SchroedingersChinchilla** [reply]
PS, I can't write to /usr/ on our computing cluster, so if I need to bind e.g. $PATH or what not differently, that would also be something I'd appreciate knowing.

**Reply by haneug** [reply]
Sebastian explained this nicely in this issue:
https://github.com/grimme-lab/xtb-python/issues/56#issuecomment-898300692

**Reply by SchroedingersChinchilla** [reply]
Great!  That's very helpful, thank you!
